pdns/packethandler.cc

```diff
@@ -71,14 +71,15 @@ PacketHandler::PacketHandler():B(g_programname), d_dk(&B)
   d_doExpandALIAS = ::arg().mustDo("expand-alias");
   d_logDNSDetails= ::arg().mustDo("log-dns-details");
   string fname= ::arg()["lua-prequery-script"];
+
   if(fname.empty())
   {
     d_pdl = nullptr;
   }
   else
   {
-    d_pdl = std::make_unique<AuthLua4>();
-    d_pdl->loadFile(fname); // XXX exception handling?
+    d_pdl = std::make_unique<AuthLua4>(::arg()["lua-global-include-dir"]);
+    d_pdl->loadFile(fname);
```

Comment:
why is the comment gone?

Reply:
turns out the exception handling is just fine.